### PR TITLE
front: open-data import: keep stop at end of path

### LINKED ISSUE #8541: No more stop at the end of open data imported trains
### What happened?

I imported trains between RES and QR and none of them as a stop at the end. This is due to the fact that this stop used to be automatically added at each circulation but it's not the case anymore.

### What did you expect to happen?

I'm expecting the impported trains to trop at the end.

### How can we reproduce it (as minimally and precisely as possible)?

1. Import open data trains

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Google Chrome

### OSRD version (top right corner `Account` button > `Informations`)

c885d29

### PR DESCRIPTION
**Pull request by Castavo**
close #8541

### CODE REVIEW

**Comment by emersion** [comment]
Ah. Is this something the front is supposed to do, or is this something the backend is supposed to handle?

**Comment by Castavo** [comment]
@emersion If you don't specify a stop at the end of a path, then the simulated train won't stop. It's the expected behavior of the backend.
You need to have the stop (even for 0s) at the end of the path if you want the train to stop

File: front/src/modules/trainschedule/components/ImportTrainSchedule/generateTrainSchedulesPayloads.ts [hunk 1]
```diff
@@ -20,12 +20,15 @@ export function generateV2TrainSchedulesPayloads(
...
+          const schedulePoint = {
             at: stepId,
-            arrival: isoDuration,
-            ...(step.duration ? { stop_for: `PT${step.duration}S` } : {}),
-          });
+            arrival: formatDurationAsISO8601(timeDifferenceInSeconds),
+            stop_for: null as string | null,
+          };
+          if (step.duration || index === train.steps.length - 1) {
+            schedulePoint.stop_for = `PT${step.duration}S`;
```
Comment (clarani) [inline front/src/modules/trainschedule/components/ImportTrainSchedule/generateTrainSchedulesPayloads.ts]: You should set the value 0 by default, for if the step.duration is undefined ;)
```suggestion
            schedulePoint.stop_for = `PT${step.duration || 0}S`;
```

**Review by RomainValls — APPROVED** [review]
LGTM and tested ✅

File: front/src/modules/trainschedule/components/ImportTrainSchedule/generateTrainSchedulesPayloads.ts (hunk 1 above)
Comment (Castavo) [reply front/src/modules/trainschedule/components/ImportTrainSchedule/generateTrainSchedulesPayloads.ts]: Indeed

**Review by clarani — APPROVED** [review]
LGTM & tested :white_check_mark:

**Pull request merged**